telemetry/definitions/commonDefinitions.json

```diff
@@ -3,7 +3,7 @@
         {
             "name": "action",
             "type": "string",
-            "description": "Name of an action that was taken, displayed, etc."
+            "description": "Name of an action that was taken, displayed, etc. See also `userChoice`."
```

Comment:
It's important to connect things with cross-references. That helps discovery, helps avoid duplicate concepts, etc.